src/Microsoft.AspNetCore.Identity.EntityFrameworkCore/IdentityEntityFrameworkBuilderExtensions.cs

```diff
@@ -30,32 +30,68 @@ public static IdentityBuilder AddEntityFrameworkStores<TContext>(this IdentityBu
 
         private static void AddStores(IServiceCollection services, Type userType, Type roleType, Type contextType)
```

Comment:
This is the important method to review, as its doing the mega type inference we need to do for the custom poco scenario